.github/workflows/test.yml

```diff
@@ -31,10 +31,11 @@ jobs:
             go.mod
             go.sum
       - name: Test and Create Coverage Report
+        # FIXME: make test-unit-cover fails 
```

Comment:
@MalteHerrmann FYI

Reply:
@fedekunze This is fixed on the base branch